learning_cp/eternity2/features3.jl

```diff
@@ -0,0 +1,72 @@
+struct EternityFeaturization <: SeaPearl.AbstractFeaturization end
```

Comment:
Eternity2 features files should be entirely reworked:

- Why having 3 different files with 3 different featurization in the first one?
- Many comments and explanations are missing;
- As previously, I dont think we should have many different featurizations on `master`, only the best one. The others are for us, for testing purpose.

Reply:
You are right. I kept the features3 which gives features to the value nodes, and works with FullFeaturedCPNN. I commented the code.